docs/details/base-widget/scroll.rst

```diff
@@ -74,8 +74,23 @@ can be set directly on the :cpp:enumerator:`LV_PART_SCROLLBAR` part of a Widget
 the Widget's or any parent's main part to make a scrollbar inherit the
 base direction.
 
-``pad_left/right/top/bottom`` sets the spacing around the scrollbars and
-``width`` sets the scrollbar's width.
+``pad_left/right/top/bottom`` sets the spacing around the scrollbars,
+``width`` sets the scrollbar's width and ``length`` sets the scrollbar's length:
+
```

Comment:
Minor suggestion: 
```suggestion
If `length` is not set or left at `0` the scrollbar's length will be set automatically according to the length of the content.  
```

Reply:
Thanks, I'll commit that.

I also wanted to add a note regarding the minimum length but I had some issues; right now the minimum length is defined as SCROLLBAR_MIN_SIZE which is an alias for LV_DPX(10) so the actual minimum length in pixels depends on the display's DPI setting, which makes it a bit hard to comment in the context of such a specific style setting, given that AFAIK the concept of DPX is not explained in the docs (yet).

Reply:
True, the length units (px, %, dpx) are not documented yet. 
cc @vwheeler63 Can you imagine a good place for these in the docs? 

Besides these I'm thinking about adding `em` (same as in CSS, meaning 1 font line height unit) and `aspect_ratio` (e.g se the width proportionally to the height)

Reply:
> True, the length units (px, %, dpx) are not documented yet.

px and % are documented [here](https://docs.lvgl.io/master/details/base-widget/coord.html#units), but DPX isn't anywhere, but that's definitely where it should be -- another bullet item.  :-)

Reply:
> Besides these I'm thinking about adding `em` (same as in CSS, meaning 1 font line height unit)

If you do that then "points" (pt) is the next logical unit (1/72nd of an inch).  (I was a typesetter for a while, so I use this a lot, but TBH, have not found a need for it yet in my firmware projects.  I mention it in case others have requested it.)